src/Fable.Cli/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+- [Python] Use `Any` type for all non-repeated generic arguments (by @dbrattli)
+- [Python] Don't generate unnecessary type type-vars if generic type is replaced by `Any` (by @dbrattli)
+- [Python] Generate new style `_T | None` instead of `Optional[_T]` (by @dbrattli)
```

Comment:
Hello @dbrattli,

The changelog doesn't use `[language] ...` format anymore (was a bad/incomplete solution on my side)

I added support deeper level of nesting in our changelog parser to support:

```md
### Added

#### JavaScript

* Item 1
* Item 2

#### Python

* Item 1

### Fixed

#### Python

* .....
```

The idea is to make the changelog structured and easier to read. You can find the category (Added, Fixed, Changed, etc.) first and then the language impacted (JavaScript, Python, Rust, All).

I based this structure on https://github.com/godotengine/godot/blob/dcbee437f7f238801e0c1a19a12a8d0e0aec63b4/CHANGELOG.md which has similar needs as us in term of formatting and they seems pretty successful with it.

Reply:
Thanks for the heads-up @MangelMaxime. I will fix!